src/Components/Cart.jsx

```diff
@@ -3,6 +3,10 @@ import { XIcon } from "@heroicons/react/outline";
 import React, { Fragment } from "react";
 
 export default function Cart({ open, setOpen, cart, updateCart }) {
+    var subtotal = 0;
```

Comment:
you can use `array.reduce` method to calculate the subtotal and store it in a `const` since we would not update it